src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -4,16 +4,15 @@
     using System.Threading.Tasks;
     using AcceptanceTesting.Support;
 
-    public class DefaultServer : IEndpointSetupTemplate
+    public class DefaultServer : ExternallyManagedContainerServer
```

Comment:
Why inheriting from `ExternallyManagedContainerServer`?

Reply:
because that has been made the actual implementation of the "default server" which contains all the configuration. The default server is a subclass from that which extends it to a "managed container server". (we should probably rename the `ExternallyManagedContainerServer` to `ExternallyManagedContainerDefaultServer`)

Reply:
Isn't it more "EndpointWhereTheContainerModeIsNotDecidedYet" ?

Reply:
Well, both of these inherit from `IEndpointSetupTemplate` so maybe:
`DefaultEndpointTemplate` and `ManagedContainerEndpointTemplate` (for the one that uses container managed by NSB via UseContainer) ?

Reply:
`DefaultServer` has been the standard name for a long while and changing that will impact 99% of all acceptance tests. I'd prefer to not make that change in this PR.

Reply:
Yeah, you're right. Let's keep it.